templates/default/hive-site.xml.erb

```diff
@@ -150,7 +150,7 @@
   </property>
   <property>
     <name>hive.exec.reducers.bytes.per.reducer</name>
-    <value>256000000</value>
+    <value>128000000</value>
```

Comment:
This controls the size of task in reducer. Setting a smaller value will increase number of tasks of reducer.